website/content/docs/configure.mdx

```diff
@@ -80,8 +80,8 @@ each can be found below:
   [Packer's home directory](#packer-s-home-directory) for more.
 
 - `PACKER_GITHUB_API_TOKEN` - When using Packer init on HCL2 templates, Packer
-  queries the public API from Github which limits the amount of queries on can
-  set the `PACKER_GITHUB_API_TOKEN` with a Github Token to make it higher.
+  queries the public API from GitHub which limits the amount of queries on can
+  set the `PACKER_GITHUB_API_TOKEN` with a GitHub Token to make it higher.
```

Comment:
Already present before but `queries on can set` reads weird, looks like we're missing some parts to this sentence.
```suggestion
  queries the public API from GitHub which has rate-limits on its APIs.
  You can set the `PACKER_GITHUB_API_TOKEN` environment variable
  with a GitHub Token to make it higher.
```